renderer-macos/lib/stubs/BriskApplication.c

```diff
@@ -0,0 +1,18 @@
+#import "BriskApplicationDelegate.h"
```

Comment:
I think this file could be called `NSApplication` since it doesn't create an application subclass.

Reply:
But it also doesn't provide all the bindings to `NSApplication`, this way, we avoid confusion as to why there's a bare minimum of methods - it's our stub and doesn't intend to fully map to the real `NSApplication`.

Reply:
fair enough.